project/Project.scala

```diff
@@ -17,9 +17,11 @@
 import io.zipkin.sbt.{BuildProperties,PackageDist,GitProject}
 import sbt._
 import Keys._
+import bintray.BintrayKeys._
 
 object Zipkin extends Build {
-  val zipkinVersion = "1.2.0-SNAPSHOT"
+
+  val zipkinVersion = "1.2.0-ALPHA"
```

Comment:
This should stay snapshot. The release number should be taken from the branch name.